docs/source/faq(developers).md

```diff
@@ -258,3 +258,12 @@ rm -rf bower_components
 npm install
 bower install
 ```
+
+#### Q. While trying to build EvalAI and run the evalai_django docker image: 
```

Comment:
`Q. While trying to build EvalAI and run the evalai_django docker image:` -> Q. While trying to build EvalAI from the master branch and run the command `docker-compose up`: